interop-testing/src/test/java/io/grpc/testing/integration/StressTestClientTest.java

```diff
@@ -108,58 +111,112 @@ public void allCommandlineSwitchesAreSupported() {
     assertEquals(60, client.metricsLogRateSecs());
   }
 
+  @Test
+  public void gaugesShouldBeExported_ipv4() throws Exception {
+    checkGaugesShouldBeExported(IPV4);
```

Comment:
:frowning_face:, do we really need to check the gauges for all the different IP binding methods?

Is there any document describing why we are messing with the stress client?

Reply:
Because it is the only test that actually talks to a server over IP.

Reply:
So make a new test? I don't follow. It doesn't even need a client; you could just open a TCP connection to see if the port is open, if you wanted.

What clued me in to this is the switch statement. We really want to avoid logic in tests.

Reply:
done